tests/behaviour/contracts/expressions/literalExpressionEvaluator.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.8.10;
+
+//SPDX-License-Identifier: MIT
+
+contract WARP {
+  function false_and() pure public returns (bool) {
+    return true && false;  // false 
+  }
+
+  function true_or() pure public returns (bool) {
+    return false || true;  // true 
+  }
```

Comment:
None of these functions are testing the behavior you just implemented. Notice that for your logic to be executed one of the members of the binary operation must return `null` in its evaluation, meaning that it was not possible to know its value during compilation time. In this case both members are boolean literals, so none of them is going to return `null` in that evaluation